<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_options_widget_base</class>
 <widget class="QWidget" name="table_options_widget_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Output Table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_extended_output">
        <property name="text">
         <string>Extended (user customizable)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="output_table_options_button">
        <property name="text">
         <string>Extended Table Optionsâ€¦</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="radio_basic_output">
        <property name="text">
         <string>Basic (default FDSN text format)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
